### PR TITLE
remove empty anchor

### PR DESCRIPTION
**Pull request by cvazac**
screen shot 2018-01-16 at 1 57 44 pm

***
<a href="https://pr-preview.s3.amazonaws.com/cvazac/server-timing/pull/50.html" title="Last updated on Jan 17, 2018, 3:06 PM GMT (c65649d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/server-timing/50/8d57aa1...cvazac:c65649d.html" title="Last updated on Jan 17, 2018, 3:06 PM GMT (c65649d)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -184,7 +184,7 @@
...
+  The interfaces defined in this specification expose potentially sensitive application and infrastructure information to any web page that has included a resource that advertises server timing metrics. For this reason the access to `PerformanceServerTiming` interface is restricted by the <a data-cite="!HTML5#same-origin">same origin</a> policy by default. Resource providers can explicitly allow server timing information to be available by adding the `Timing-Allow-Origin` HTTP response header, as defined in [[!RESOURCE-TIMING-2]], that species the domains that are allowed to access the server metrics.
```
Comment (yoavweiss) [inline index.html]: s/species/specifies/

**Comment by cvazac** [comment]
@igrigorik It was certainly meant to link to _something_, but [https://w3c.github.io/resource-timing/#cross-origin-resources](https://w3c.github.io/resource-timing/#cross-origin-resources) didn't fit for me. The server timing bits tell me `.serverTiming` shouldn't be exposed on ResourceTimingEntry objects. The link refers to durations being zeroed out. Or maybe the resource-timing bits should be updated *and* fix link.

**Comment by yoavweiss** [comment]
Yeah, the behavior we ended up with for crossorigin server timing with no TAO seems significantly different than what we had for ResourceTiming.

**Comment by igrigorik** [comment]
> Yeah, the behavior we ended up with for crossorigin server timing with no TAO seems significantly different than what we had for ResourceTiming.

I'm confused.. Per https://w3c.github.io/server-timing/#processing-model, we talk about doing a check against timing-allow-origin algorithm. So, it's same as RT — correct? That said, re-reading this particular update, I agree that cross-origin link is probably unnecessary.

**Comment by cvazac** [comment]
As the spec reads today, a cross origin resource will have: 
```
{
  ...
  serverTiming: []
}
```

Previous versions of the spec had "zeroed out" entries:
```
{
  ...
  serverTiming: [{
    name: 'blah', 
    duration: 0, 
    description: ''
  }, ... ]
}
```

**Pull request merged**